resources/views/hardware/labels.blade.php

```diff
@@ -118,6 +118,11 @@
             <br>
         @endif
         </div>
+	<div class="pull-left">
+        @if (($settings->labels_display_company_name=='1') && ($asset->company))
+        	C: {{ $asset->company->name }}
+        @endif
+    	</div>
```

Comment:
One more thing - do we want the empty divs if company support isn't enabled on labels? Maybe:

```
@if (($settings->labels_display_company_name=='1') && ($asset->company))
<div class="pull-left">
     C: {{ $asset->company->name }}
</div>
@endif
```

I'm honestly not sure - the labels are pretty cramped anyway, so I'm *guessing* we wouldn't want it, but def open to discussion.

Reply:
Valid point, if so should this be the same on the other optional fields?